docs/run-command.md

```diff
@@ -9,11 +9,18 @@ of running scripts in an Azure Linux VM also apply here.
 ## Regardless of the Kubernetes control plane status
 
 When executing `run-command` without passing through Kubernetes, we need to provide the information of the node (VMSS instance) where we want to run the given command. To retrieve such information, we can use the [`config import`](./config.md#importing-configuration) command. Once we got it, we can select the node we want to use and all the subsequent `run-command` commands will be executed on that node:
-
 ```bash
+# Clean the current configuration (if any)
+kubectl aks config unset-all
 # Import the nodes information with the cluster information
-$ kubectl aks config import --subscription mySubID --resource-group myRG --cluster-name myCluster
-$ kubectl aks config show
+kubectl aks config import --subscription $mySubID --resource-group $myRG --cluster-name $myCluster
```

Comment:
We are using in some places `$mySubID` and in some other `$SUBSCRIPTION`. I think we should align them and it should be with the latest and define them as per previous comment.